buildkite/src/Jobs/Unit/Pipeline.dhall

```diff
@@ -0,0 +1,41 @@
+let Prelude = ../../../External/Prelude.dhall
+
+let Decorate = ../../../Lib/Decorate.dhall
+
+let Pipeline = ../../../Pipeline/Dsl.dhall
+let Command = ../../../Command/Base.dhall
+let Docker = ../../../Command/Docker/Type.dhall
+let Size = ../../../Command/Size.dhall
+let JobSpec = ../../../Pipeline/JobSpec.dhall
+
+let commands =
+  [
+    "./scripts/skip_if_only_frontend_or_rfcs.sh bash -c 'source ~/.profile && (dune runtest src/nonconsensus --profile=nonconsensus_medium_curves -j8 || (./scripts/link-coredumps.sh && false))'",
```

Comment:
You can rip out `skip_if_only_frontend_or_rfcs`, this should be covered by the spec `dirtyWhen` instead.

Reply:
Ah nice, done